modular_skyrat/modules/assault_operatives/code/dynamic_rulsesets_roundstart.dm

```diff
@@ -0,0 +1,64 @@
+//////////////////////////////////////////////
+//                                          //
+//          ASSAULT OPERATIVES              //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/assault_operatives
+	name = "Assault Operatives"
+	antag_flag = ROLE_ASSAULT_OPERATIVE
+	antag_datum = /datum/antagonist/assault_operative
+	minimum_required_age = 14
+	restricted_roles = list(
+		JOB_CAPTAIN,
+		JOB_HEAD_OF_SECURITY,
+	)
+	required_candidates = 5
+	weight = 3
+	cost = 20
```

Comment:
That cost feels incredibly low, but I'd need to see how much chaos they end up causing, to really be able to tell.

Reply:
Should be fine to start